src/sw/redis++/async_redis.h

```diff
@@ -1565,6 +1565,32 @@ class AsyncRedis {
         return zrevrangebylex<Output, Interval, Callback>(key, interval, {}, std::forward<Callback>(cb));
     }
 
+    // TODO: withscores parameter
+    template <typename Output, typename Interval>
+    Future<Output> zrevrangebyscore(const StringView &key,
+                                    const Interval &interval,
+                                    const LimitOptions &opts) {
+        return _command<Output>(fmt::zrevrangebyscore<Interval>,
+                                key, interval, opts);
+    }
+
+    template <typename Output, typename Interval, typename Callback>
+    auto zrevrangebyscore(const StringView &key, const Interval &interval, const LimitOptions &opts, Callback &&cb)
+        -> typename std::enable_if<IsInvocable<typename std::decay<Callback>::type, Future<Output> &&>::value, void>::type {
+        _callback_fmt_command<Output>(std::forward<Callback>(cb), fmt::zrevrangebyscore<Interval>, key, interval, opts);
+    }
+
+    template <typename Output, typename Interval>
+    Future<Output> zrevrangebyscore(const StringView &key, const Interval &interval) {
+        return zrevrangebyscore(key, interval, {});
```

Comment:
We need specify template parameter for calling zrevrangebyscore:

```
return zrevrangebyscore<Output>(key, interval, {});
```

Reply:
done~